client_src/js/lib/modules/currentOperations.js

```diff
@@ -43,7 +43,6 @@ export default function currentOperations(
     });
   }
 
-  currentOperationsCount = currentOperationsCount;
```

Comment:
It does me make me wonder when I see stuff like this bahaha

Reply:
JUST ASSIGN IT DAMMIT!